ajax/map.php

```diff
@@ -69,15 +69,34 @@
    $points = [];
    foreach ($rows as $row) {
       $idx = $row['raw']["ITEM_$lat_field"] . ',' . $row['raw']["ITEM_$lng_field"];
+      $level = count(explode(' > ', $row['raw']["ITEM_$name_field"]));
       if (isset($points[$idx])) {
          $points[$idx]['count'] += 1;
+         if ($level < $points[$idx]['level']) {
+            // It's the higher level
+            $points[$idx]['level'] = $level;
+            $points[$idx]['title'] = $row['raw']["ITEM_$name_field"];
+            $points[$idx]['loc_ids'] = [$row['raw']['loc_id']];
```

Comment:
`loc_ids` does not seems used

Reply:
It's used to define if the location name if yet added in title.
The behavior is:
* if the location level is higher, it replace the name and the id in _loc_ids_ list
* if the location level is same, it check if the id in _loc_ids_ list, if not it add the name to the previous title (so will have many locations names in same location level)

Reply:
Ok, so this is used  locally only, but sent along with the ajax result. I think this is useless to return it since this is not used ;)

Reply:
You're right, I will modify it

Reply:
I add code to fix that in a new commit, not squashed to see the diff